README.md

```diff
@@ -1,26 +1,208 @@
+[Semantic Versioning][https://semver.org].
+
+For example, if a module has versions `5.2.0-rc1`, `5.1.1`, `5.1.0`, `5.1.0-rc1`, `5.1.0-beta1`, `5.0.0`, `5.0.0-rc1`,
+`4.10.1`, `4.10.0`, and `4.9.0`:
+
+* `5.*` would pin the module to the latest release of version 5, `5.1.1` (prerelease versions are ignored).
+* `4.10.*` would pin the module to the latest prerelease of version `4.10`, `4.10.1` (prerelease versions are ignored).
+* `5.*-rc*` would pin the module to the latest release or prerelease of version 5, `5.2.0-rc1`. In order to use
+prerelease versions, the version *must* contain a `-` prerelease prefix.
+
+## Path
+
+The `Path` property is optional. By default, the module is saved to a `PSModules` directory in the same directory as
+the "pxget.json" file. If the `Path` property is provided, the module will instead be saved to that directory, which
+will be created if it doesn't exist. Relative paths are resolved from the directory of the "pxget.json" file.
+
+
+# Using Private Modules
+
+## Importing
+
+To use a private module installed by PxGet, use `Import-Module` and pass the path to the module instead of a module
+name. Use `Join-Path` and join the `$PSScriptRoot` automatic variable—the path to the current script's directory—with
+the relative path to the module in the PSModules directory.
+
+```powershell
+# If a script is in the same directory as the "PSModules" directory.
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath 'PSModules\Whiskey' -Resolve)
+
+# If the script is in a sub-directory.
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\Whiskey' -Resolve)
+```
+
+## Best Practices
+
+### In Scripts
+
+***DO*** always use the `Import-Module` cmdlet's `Alias`, `Cmdlet`, and `Function` parameters to explicitly list what
+commands your script is importing and using. It makes upgrading easier when you know what commands you're using.
+
+***DO NOT*** depend on PowerShell's automatic module loading. That functionality won't see the private modules in
+"PSModules".
+
+### When Writing Modules
+
+***DO*** ship your module's dependencies as nested modules, preferably in a "Modules" directory. Import dependencies
+from that private location. A module can have its own version of a module loaded privately.
+
+***DO NOT*** use the `NestedModules` module manifest property. Use an explicit `Import-Module` in your root module to
+import dependencies saved inside your module.
+
+***DO NOT*** use the `TypesToProcess` module manifest property, i.e. don't specify extended type data in a .ps1xml file.
+PowerShell writes errors and refuses to import a module if its type file has previously been loaded. Instead, in your
+root module, check if members are present on types, and add them using the `Update-TypeData` cmdlet if they are not
+present.
+
+***TRY NOT*** to use or load private assemblies. Only one version of an assembly can be loaded at a time, and once an

```

Comment:
For the less informed who may not know what we mean by "assemblies", maybe call out that this means `*.dll` files.